doc/coloring/coloring-family.rst

```diff
@@ -7,6 +7,13 @@
     Alike 3.0 License: https://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+* **Supported versions:**
+  `Latest <https://docs.pgrouting.org/latest/en/coloring-family.html>`__
+  (`3.2 <https://docs.pgrouting.org/3.2/en/coloring-family.html>`__)
+
```

Comment:
Add this link to the non existing links in conf.py.in **in this branch** do not do the change on the website, do it in your sublime
```
https://docs.pgrouting.org/latest/en/coloring-family.html
```